src/story-utils/decorators.js

```diff
@@ -29,3 +30,19 @@ export const RouterDecorator = (Story, {args: {routerArgs = {}}}) => {
     </MemoryRouter>
   );
 };
+
+export const FormikDecorator = (Story, context) => {
+  const {initialValues} = context.parameters;
+
+  return (
+    <Formik
+      initialValues={initialValues}
+      enableReinitialize
+      onSubmit={(values, formikHelpers) => console.log(values, formikHelpers)}
+    >
+      <form>
+        <Story {...context} />
```

Comment:
```suggestion
        <Story />
```
Is this prop spreading required? I've personally never needed it so I'm also unsure what exactly it does.